apps/docs/src/app/platform/component-docs/platform-forms/platform-textarea/platform-textarea-examples/platform-textarea-basic-example.component.html

```diff
@@ -0,0 +1,95 @@
+    <div *ngIf="true" class="fd-page__content fd-has-background-color-neutral-2">
+        <fdp-form-group #fg1 [multiLayout]="true" [formGroup]="form" [object]="data">
+            <fdp-form-field
+                #ff1
+                [id]="'basicTextarea'"
+                label="Basic Textarea with Platform Forms"
+                [placeholder]="'Start entering detailed description'"
+                hint="This is tooltip help"
+                zone="zLeft"
+                rank="10"
+                hintPlacement="'left'"
+            >
+                <fdp-textarea [name]="'basicTextarea'" [formControl]="ff1.formControl"></fdp-textarea>
+            </fdp-form-field>
+
+            <fdp-form-field
+                #ff2
+                [id]="'readonlyDescription'"
+                label="Readonly Textarea"
+                [placeholder]="'Start entering detailed description'"
+                hint="This is tooltip help"
+                zone="zLeft"
+                rank="10"
+                hintPlacement="'left'"
+            >
+                <fdp-textarea [name]="'readonlyDescription'" [formControl]="ff2.formControl" [readonly]="true">
+                </fdp-textarea>
+            </fdp-form-field>
+
+            <!-- <fdp-form-field
+                #ff3
+                [id]="'disabledDescription'"
+                label="Disabled Textarea"
+                [placeholder]="'Start entering detailed description'"
+                hint="This is tooltip help"
+                zone="zLeft"
+                rank="10"
+                hintPlacement="'left'"
+                
+            >
+                <fdp-textarea formControlName="ff3" > </fdp-textarea>
+            </fdp-form-field> -->
+            <fdp-form-field
+                #ff3
+                [id]="'disabledDescription'"
+                label="Disabled Textarea"
+                [placeholder]="'Start entering detailed description'"
+                hint="This is tooltip help"
+                zone="zLeft"
+                rank="10"
+                hintPlacement="'left'"
+            >
+                <fdp-textarea [name]="'readonlyDescription'" [disabled]="true" [value]="'this is a disabled textarea'">
+                </fdp-textarea>
+            </fdp-form-field>
+
+            <fdp-form-field
+                #ff4
+                [id]="'compactTextarea'"
+                label="Compact Textarea"
+                [placeholder]="'Start entering detailed description'"
+                hint="This is tooltip help"
+                zone="zLeft"
+                rank="10"
+                hintPlacement="'left'"
+            >
+                <fdp-textarea [name]="'compactTextarea'" [formControl]="ff4.formControl" [contentDensity]="'compact'">
+                </fdp-textarea>
+            </fdp-form-field>
+        </fdp-form-group>
+
+        <div fd-form-item>
+            <label fd-form-label for="textarea-1">Textarea without Platform Forms</label>
```

Comment:
I think it should be clarified what a 'platofrm form' is

Reply:
Basically when we use components in the context of `fdp-form-field` and `fdp-form-group`, we say we use it with Platform Forms. 
@fkolar are we talking about Platform Forms in the homepage of Platform documentation pages?